src/ZString.Unity/Assets/Scripts/ZString/FormatHelper.cs

```diff
@@ -48,17 +48,17 @@ private static void FormatToRightJustify<TBufferWriter, T>(ref TBufferWriter sb,
             if (typeof(T) == typeof(string))
             {
                 var s = Unsafe.As<string>(arg);
-                int padding = width - s.Length;
+                int padding = width - s!.Length;
                 if (padding > 0)
                 {
                     var paddingSpan = sb.GetSpan(padding);
                     paddingSpan.Fill(sp);
                     sb.Advance(padding);
                 }
 
-                var span = sb.GetSpan(s.Length);
-                s.AsSpan().CopyTo(span);
-                sb.Advance(s.Length);
+                var span = sb.GetSpan(s!.Length);
+                s!.AsSpan().CopyTo(span);
```

Comment:
Redundant when the initialization of `s` is adjusted - `var s = Unsafe.As<string>(arg)!;`